### PR TITLE
feat(atomic)!: meta update + major bump

### PR DESCRIPTION
**Pull request by louis-bompart**
## Proposed changes

Update some meta in the ACCD packages.

Will be merged with footer: 
`BREAKING CHANGE: Official release of the Atomic Custom Components Distribution Packages.`

## Breaking changes

No true breaking

### CODE REVIEW

**Review by y-lakhdar — APPROVED** [review]

File: packages/ui/atomic/create-atomic-component-project/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "test": "node --experimental-vm-modules --no-warnings ../../../../node_modules/jest/bin/jest.js"
   },
   "keywords": [],
```
Comment (mrrajamanickam-coveo) [inline packages/ui/atomic/create-atomic-component-project/package.json:18]: add keywords here for consistency?

File: packages/ui/atomic/create-atomic-component/package.json [hunk 2]
```diff
@@ -15,8 +15,11 @@
     "release:phase1": "npx -p=@coveord/release npm-publish",
     "test": "node --experimental-vm-modules --no-warnings ../../../../node_modules/jest/bin/jest.js"
   },
-  "keywords": [],
-  "author": "",
+  "keywords": [
+    "coveo",
```
Comment (mrrajamanickam-coveo) [inline packages/ui/atomic/create-atomic-component/package.json:19]: add `cli` in addition to or `coveo-cli` instead of `coveo` ?

Comment (louis-bompart) [reply packages/ui/atomic/create-atomic-component/package.json:19]:
```suggestion
    "coveo",
    "cli",
```

File: packages/ui/atomic/create-atomic-result-component/package.json [hunk 3]
```diff
@@ -15,8 +15,11 @@
     "release:phase1": "npx -p=@coveord/release npm-publish",
     "test": "node --experimental-vm-modules --no-warnings ../../../../node_modules/jest/bin/jest.js"
   },
-  "keywords": [],
-  "author": "",
+  "keywords": [
+    "coveo",
```
Comment (louis-bompart) [inline packages/ui/atomic/create-atomic-result-component/package.json:19]:
```suggestion
    "coveo",
    "cli",
```

File: packages/ui/atomic/create-atomic-component-project/package.json (hunk 1 above)
Comment (louis-bompart) [reply packages/ui/atomic/create-atomic-component-project/package.json:18]:
```suggestion
  "keywords": [
    "coveo",
    "cli",
    "atomic"
  ],
```

**Pull request merged**